crates/sui-e2e-tests/tests/shared_objects_tests.rs

```diff
@@ -330,237 +269,104 @@ async fn access_clock_object_test() {
     }
 }
 
-/// Same test as `call_shared_object_contract` but the clients submits many times the same
-/// transaction (one copy per authority).
-#[sim_test]
-async fn shared_object_flood() {
```

Comment:
Why is this test removed?

Reply:
It doesn't test anything that call_shared_object_contract hasn't